docs/custom-flows/add-phone.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Add phone to account
+description: Learn how to create a UI component for users to add phone number to their account
+---
+
+# Add phone to account
+
```

Comment:
I feel like it would be good to have at least a brief description before the big code block here, even if its cursory

Reply:
We're definitely going to. I am checking with @alexisintech to see what parts she wants to handle, if any. I created 4 PRs today and all will stay in Draft until the info is written.